go/pkg/client/tree_test.go

```diff
@@ -348,6 +349,40 @@ func TestComputeMerkleTreeEmptyStructureVirtualInputs(t *testing.T) {
 	}
 }
 
+func TestComputeMerkleTreeEmptyRoot(t *testing.T) {
+	root, err := ioutil.TempDir("", t.Name())
+	if err != nil {
+		t.Fatalf("failed to make temp dir: %v", err)
+	}
+	defer os.RemoveAll(root)
+	inputSpec := &command.InputSpec{
+		Inputs: []string{"."},
+	}
+	cache := newCallCountingMetadataCache(root, t)
+
+	e, cleanup := fakes.NewTestEnv(t)
+	defer cleanup()
+
+	gotRootDg, inputs, stats, err := e.Client.GrpcClient.ComputeMerkleTree(root, inputSpec, cache)
+	if err != nil {
+		t.Errorf("ComputeMerkleTree(...) = gave error %v, want success", err)
+	}
+	if diff := cmp.Diff(digest.Empty, gotRootDg); diff != "" {
+		t.Errorf("ComputeMerkleTree(...) gave diff (-want +got) on root:\n%s", diff)
+	}
+	if len(inputs) != 1 {
+		t.Errorf("ComputeMerkleTree(...) should only include one input:\n%v", inputs)
+	}
+	wantInput := uploadinfo.EntryFromBlob([]byte{})
```

Comment:
```suggestion
	wantInput := uploadinfo.EntryFromBlob(nil)
```

Reply:
It seems to break cmp.Diff().
https://buildkite.com/bazel/remote-apis-sdks/builds/2259#4da0f44e-8cfa-4271-b39e-782f7f0054d9/11108-11113